### ISSUE TITLE
Formatting issue occurs when a dollar sign is included in the failure message

### ISSUE THREAD

**Issue by DMEvans** [scssyworks/ng-lint-report#1]
I have come across an issue when running this against a JSON lint report generated from an Angular project.  I've trimmed it down to the following example where a dollar sign has been included in the failure message.  Here is the JSON:

`[{"endPosition":{"character":16,"line":13,"position":468},"failure":"expected member-variable-declaration: 'config$' to have a typedef","name":"C:/TestCode/config.service.ts","ruleName":"typedef","ruleSeverity":"ERROR","startPosition":{"character":9,"line":13,"position":461}}]`

Here is a screen grab of the issue it causes when viewing the HTML report in a browser:
![ts-lint-html-report-issue](https://user-images.githubusercontent.com/8655991/82343860-43ad0c00-99eb-11ea-8b2c-cb0cb7b05784.PNG)

**Comment by scssyworks** [comment]
Thank you @DMEvans for raising this issue. I will have a look at it and provide an update soon.

**Comment by DMEvans** [comment]
@scssyworks - I tried to see if I could find an easy fix for this in the index.js, but everything I tried didn't seem to work.  I couldn't find any way to escape the dollar sign.

The only success I had was to strip the dollar sign from the text entirely.  
![image](https://user-images.githubusercontent.com/8655991/82550702-e03fed00-9b56-11ea-8a54-373c6f6a3c3f.png)

Hopefully, you'll be able to find a better solution.  Thanks :)
